akka-docs/rst/java/persistence.rst

```diff
@@ -618,14 +617,14 @@ the destination actor. When recovering, messages will be buffered until they hav
 Once recovery has completed, if there are outstanding messages that have not been confirmed (during the message replay),
 the persistent actor will resend these before sending any other messages.
 
-Deliver requires a ``deliveryIdToMessage`` function to pass the provided ``deliveryId`` into the message so that correlation
+Deliver requires a ``deliveryIdToMessage`` function to pass the provided ``deliveryId`` into the message so that the correlation
 between ``deliver`` and ``confirmDelivery`` is possible. The ``deliveryId`` must do the round trip. Upon receipt
-of the message, destination actor will send the same``deliveryId`` wrapped in a confirmation message back to the sender.
-The sender will then use it to call ``confirmDelivery`` method to complete delivery routine.
+of the message, the destination actor will send the same``deliveryId`` wrapped in a confirmation message back to the sender.
+The sender will then use it to call the ``confirmDelivery`` method to complete the delivery routine.
 
 .. includecode:: code/docs/persistence/PersistenceDocTest.java#at-least-once-example
 
-The ``deliveryId`` generated by the persistence module is a strictly monotonically increasing sequence number
+The ``deliveryId`` generated by the persistence module is strictly a monotonically increasing sequence number
```

Comment:
“strictly monotonic” is a mathematical term, this change is incorrect (it changes the meaning of the sentence)